source/_posts/2022-02-08-android-february.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: "Android 2022.2: Local Push Notifications!"
+description: "Local Push Notifications, In-App Changelog and more!"
+date: 2022-02-09 00:00:00
+date_formatted: "February 11, 2022"
+comments: true
+author: Daniel Shokouhi
+categories: Release-Notes
+og_image: /images/blog/2022-02-android/Companion.png
+---
+
+
+![Screenshot of Companion](/images/blog/2022-02-android/Companion.png)
+
+Hey Everyone! Its time for the next Android release and we are really excited to get this one out. Lots of changes for all versions of the app, including Wear OS and Quest devices.
+
+## Breaking Changes
+
+We have a pretty big release this month with a lot of changes going on. Some of these changes are noticeable as soon as you update the app, so lets cover those first.
+
+### Local Push Notifications
+
+The first major change that we want to cover is for [Local Push Notifications]. Before this release all notifications were routed via Google. Now notifications can be delivered directly from your Home Assistant server if you have remote access setup. These notifications do not count towards the rate limit and are shown on the device immediately, everytime.
+
+So what's the breaking change? The app needs to establish and maintain a constant connection to your Home Assistant server with WebSockets. In order to do this, we have to create a persistent notification and this can be upsetting to some users.
+
+A persistent notification lets Android know the app has important ongoing tasks where we do not want the background process stopped. The purpose of this notification is identical to our Sensor Worker, that allows the app to continue sending sensor updates while in the background. Just like Sensor Worker, our WebSockets notification also has its own notification channel to allow you to customize the appearance. You are free to minimize (or silence) this notification to hide the icon from appearing in your status bar.
+
+Some users may also decide to turn off the channel. While there is nothing preventing you from turning it off, if you run into connection issues you will be asked to turn the channel back on.
```

Comment:
Always write towards the reader, which is a single person. Not a group of people.

```suggestion
You may also decide to turn off the channel. While there is nothing preventing you from turning it off, if you run into connection issues you will be asked to turn the channel back on.
```